README.md

```diff
@@ -6,12 +6,14 @@ season -> fasl
 vs
 session -> jalase
 
-I am Bani
+I am benyamin
 
 I like
 
-1-pizza
-2-loobia
-3-bastani
 
-I enjoy being here with you
+
+1-Ghorme sarz
```

Comment:
do you mean ghorme sabzi????????????????????????

Reply:
yesssssssssss